node_modules/oae-core/groupprofile/groupprofile.html

```diff
@@ -0,0 +1,42 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/groupprofile.css" />
+
+<div class="groupprofile-widget">
+    <div id="groupprofile-header-container"><!-- --></div>
+    <div id="groupprofile-description-container"><!-- --></div>
+    <div id="groupprofile-members-container"><!-- --></div>
+</div>
+
+<div id="groupprofile-header-template"><!--
+    ${header('__MSG__ABOUT__', action)}
+--></div>
+
+<div id="groupprofile-description-template"><!--
+    {if contextProfile.description}
+        <div class="groupprofile-description">${oae.api.util.security().encodeForHTMLWithLinks(contextProfile.description)}</div>
+    {elseif contextProfile.isManager}
+        <div class="groupprofile-description-empty">
+            <button class="btn btn-link group-trigger-editgroup" data-editgroup-focus="#editgroup-description">
+                __MSG__ADD_A_DESCRIPTION_TO_THIS_GROUP__
+            </button>
+        </div>
+    {/if}
+--></div>
+
+<div id="groupprofile-members-template"><!--
+    {if members.length > 0}
+        <div class="groupprofile-members">
```

Comment:
When members are added/removed, the list of members in the groupprofile widget should be updated as well